azure/integration_quickstart/src/azure_integration_quickstart/main.py

```diff
@@ -105,10 +104,7 @@ def create_app_registration_with_permissions(scopes: Iterable[Scope]) -> AppRegi
             raise AccessError(f"{str(e)}. {APP_REGISTRATION_PERMISSIONS_INSTRUCTIONS}") from e
```

Comment:
APP_REGISTRATION_PERMISSIONS_INSTRUCTIONS is somewhat duplicative with the boilerplate we now attach to all access errors - it's also more particular to this specific kind of access error. I think the best solution would be to allow an optional argument to AccessError to override the boilerplate.